integration/fuse/src/main/java/alluxio/cli/command/FuseMetricCommand.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.command;
+
+import alluxio.AlluxioURI;
+import alluxio.Constants;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.URIStatus;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.metrics.Metric;
+import alluxio.metrics.MetricsSystem;
+import alluxio.wire.FileInfo;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Gets fuse related metrics.
+ */
+@ThreadSafe
+public final class FuseMetricCommand extends AbstractFuseShellCommand {
+
+  /**
+   * @param fs filesystem instance from fuse command
+   * @param conf configuration instance from fuse command
+   */
+  public FuseMetricCommand(FileSystem fs, AlluxioConfiguration conf) {
+    super(fs, conf, "");
+  }
+
+  /**
+   * Get metric from metric system, only support numeric metric display now.
+   * @param path the path uri from fuse command
+   * @param argv the metric name
+   * @return the result of running the command
+   */
+  @Override
+  public URIStatus run(AlluxioURI path, String[] argv) throws InvalidArgumentException {
+    // Metric Key name consist of twp parts: scope and name,
+    // so the argv array has two elements, combine them to metric key.
+    if (argv.length != 2) {
+      throw new InvalidArgumentException("Invalid metric key!");
+    }
+    String metricName = argv[0] + "." + argv[1];
+    String fullMetricName = MetricsSystem.getMetricName(metricName);
+    Metric metric = MetricsSystem.getMetricValue(fullMetricName);
+    if (metric == null) {
+      throw new InvalidArgumentException(String.format("No such metric %s found in system!",
+          argv[0]));
+    }
+    long value = 0;
+    switch (metric.getMetricType()) {
+      case COUNTER:
+        value = MetricsSystem.counter(metricName).getCount();
+        break;
+      case METER:
+        value = MetricsSystem.meter(metricName).getCount();
+        break;
+      case TIMER:
+        value = MetricsSystem.timer(metricName).getCount();
+        break;
+      default:
+        throw new InvalidArgumentException("Unsupported metric type");
+    }
+    return new URIStatus(new FileInfo().setLength(value).setCompleted(true));
```

Comment:
Is there a better command that can let us output a long paragraph?
@maobaolong @yyongycy

Reply:
The returned fileInfo has no name?

Reply:
@LuQQiu I think we can use filename instead of the whole result, something like `open.count=2`